josemak25/baby-bliss
# Ft bottom tab nav

#### What does this PR do?

Add the bottom tab navigator. 

#### Description of Task to be completed?

Added and implemented the bottom navigation tab to easily navigate through the following screens:

1. The Home screen
2. The Community Screen and 
3. The Profile screen

#### How should this be manually tested?

After cloning the repo,
`CD` into it and `RUN`

javascript
git checkout ft-bottom-tab-nav

git pull origin ft-bottom-tab-nav

yarn install

yarn test

Using an emulator: Launch the app, immediately after the splash screen you should see the bottom tab navigator at the bottom of the screen.

#### Screenshots (if appropriate)

![Simulator Screen Shot - iPhone 7 - 2020-02-27 at 18 25 59](https://user-images.githubusercontent.com/12932317/75469110-b3809d00-598e-11ea-996f-b2f54f89a332.png)
